.github/pull_request_template.md/pull_request1_template.md

```diff
@@ -0,0 +1,15 @@
+<!--If you need help with you PR or if you want to to discuss something with us, join the **WalkingPal** [Discord community](https://discord.gg/Et3aQTsW6a)!-->
+* ### If you need help with you PR or if you want to to discuss something with us, join the **WalkingPal** [Discord community](https://discord.gg/Et3aQTsW6a)!
+
+   ---------------------------------------------------
+
+
+   ### Checklist : 
```

Comment:
1. In previous PR, there was just those 6 lines only. But, in this PR there is again excess lines. Please put those 6 lines back. Remove all this

2. Your folder name is "pull_request_template.md". It must be PULL_REQUEST_TEMPLATE

Reply:
> 1. In previous PR, there was just those 6 lines only. But, in this PR there is again excess lines. Please put those 6 lines back. Remove all this
> 2. Your folder name is "pull_request_template.md". It must be PULL_REQUEST_TEMPLATE

made the changes. pls check

Reply:
@ashuvssut Hey, Thanks for guiding me through this and for your support. It was great learning. 
Will hactoberfest consider this PR for me?

Reply:
yes